### PR TITLE
Fix prefer-stateless-function

### PR DESCRIPTION
**Pull request by jhonny218**
## Description

This is part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/8286

This fixes the 1 linting error for the `prefer-stateless-function` rule.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by billfienberg — APPROVED** [review]
LGTM

**Comment by rianfowler** [comment]
Screen Shot 2020-05-07 at 11 11 03
We should make sure to have an approval from all code owners before merging changes.

**Pull request merged**
